src/components/Image/index.tsx

```diff
@@ -123,8 +123,11 @@ const Image: React.FC<ImageProps> = ({
   // if url passed in directly without resized params
   if (typeof resizedImageOptions === 'undefined' || typeof resizedImageOptions[`${largeWidth}x${largeHeight}`] === 'undefined') {
     // todo: remove for prod
-    console.error(`no resized options found for url: ${url}.`);
-    console.error('Please use resized options to save money on serving bigger images than necessary. Consider using resizer content source block or adding the resizer block to content source transform.');
+    if (Fusion?.isAdmin) {
```

Comment:
don't really know what's happening here? can this be a separate pr

Reply:
Well then we both don't know what's going on.